packages/waku/src/lib/builder/build.ts

```diff
@@ -355,9 +355,10 @@ const buildClientBundle = async (
   ssr: boolean,
 ) => {
   const mainJsFile = joinPath(rootDir, config.srcDir, config.mainJs);
-  const cssAssets = serverBuildOutput.output.flatMap(({ type, fileName }) =>
-    type === 'asset' && fileName.endsWith('.css') ? [fileName] : [],
+  const nonJsAssets = serverBuildOutput.output.flatMap(({ type, fileName }) =>
```

Comment:
Can you please do the same for the SSR bundle? We need it right?

Reply:
Copying assets to ssr folder is not needed - have a look at my new tests - the leak of "private" JSON assets is tested and works as expected.

Reply:
@dai-shi can you look at the failed  typescript error - it is not shown when I run `pnpm test` local but I think it is related to the fact that I have changed the `App` component to an async function in `./e2e/fixtures/ssr-target-bundle/src/components/App.tsx` - should I disable the error in `entries.tsx`?

> Type 'Promise<typeof import("/waku/e2e/fixtures/ssr-target-bundle/src/components/App", { with: { "resolution-mode": "import" } })>' is not assignable to type 'Promise<{ default: ComponentType<any>; }>'.

Reply:
> Copying assets to ssr folder is not needed - have a look at my new tests - the leak of "private" JSON assets is tested and works as expected.

Oh, because we don't need styles for SSR? Yeah, that's my oversight.

Reply:
I have to clarify - the "private" JSON is not leaked to the public directory in your current implementation :-)